lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -92,6 +93,27 @@
 )
 
 
+def image_modifier_checker(config):
```

Comment:
"image_modifier_checker" is a noun, functions should be verbs or if boolean, sound like the condition, so maybe better
"
```
def includes_image_modification(config):
    ...

if includes_image_modification(config)

```
?